phing/build.yml

```diff
@@ -42,7 +42,7 @@ cm:
   strategy: config-split
   core:
     # The parent directory for configuration directories, relative to the docroot.
-    path: ../config
+    path: ${repo.root}/config
```

Comment:
Ah, let me try that. The commit message on that implies that it's just fixing a test, so I didn't bother patching for it :)